default/stringtables/en.txt

```diff
@@ -11519,15 +11519,16 @@ PLC_METROPOLES
 Metropoles
 
 PLC_METROPOLES_DESC
-'''Boosts the output of up to [[value METROPOLES_PLANET_LIMIT]] planets with population of at least [[value METROPOLES_MINIMUM_POPULATION]]. The highest-[[metertype METER_POPULATION]] of the qualified planets will receive, regardless of focus:
-• Increases [[metertype METER_TARGET_RESEARCH]] by [[value METROPOLES_TARGET_RESEARCH_PERPOP]] per [[metertype METER_POPULATION]].
+'''Boosts the output of up to [[value METROPOLES_PLANET_LIMIT]] planets (but no more than 1 / [[value METROPOLES_MIN_PLANETS]] or your populated planets) with [[metertype METER_POPULATION]] of at least [[value METROPOLES_MINIMUM_POPULATION]] and [[metertype METER_HAPPINESS]] of at least [[value METROPOLES_MIN_STABILITY]].
+The highest-populated of the qualified planets will receive, regardless of focus:
+• Increases [[metertype METER_TARGET_RESEARCH]] by [[value METROPOLES_TARGET_RESEARCH_PERPOP]] per population.
 • Increases [[metertype METER_TARGET_INDUSTRY]] by [[value METROPOLES_TARGET_INDUSTRY_PERPOP]] per population.
 • Increases [[metertype METER_TARGET_CONSTRUCTION]] by [[value METROPOLES_TARGET_CONSTRUCTION_FLAT]].
 • Increases [[metertype METER_MAX_SUPPLY]] by [[value METROPOLES_MAX_SUPPLY_FLAT]].
 but also has a penalty:
 • Decreases [[metertype METER_MAX_STOCKPILE]] by [[value METROPOLES_MAX_STOCKPILE_FLAT]].
 
-Additionally, other populated planets (with lower than the highest population planets, including all with less than [[value METROPOLES_MINIMUM_POPULATION]] [[metertype METER_POPULATION]]) have a penalty applied:
+Additionally, other planets (with lower than the highest population planets, including all with less than [[value METROPOLES_MINIMUM_POPULATION]] population or less than [[value METROPOLES_MIN_STABILITY]] stability) have a penalty applied:
```

Comment:
why do you remove the metertype link for population (and not add one for stability)?

Reply:
Wikipedia style: Link a word only where it appears for the first time.